simpleansi.go

```diff
@@ -18,15 +18,28 @@ func MoveCursor(row, col int) {
 
 const RESET = "\x1b[0m"
 
-var COLOURS = map[string]string{
-	"BLACK":   "\x1b[1;30;40m",
-	"RED":     "\x1b[1;31;41m",
-	"GREEN":   "\x1b[1;32;42m",
-	"BROWN":   "\x1b[1;33;43m",
-	"BLUE":    "\x1b[1;34;44m",
-	"MAGENTA": "\x1b[1;35;45m",
-	"CYAN":    "\x1b[1;36;46m",
-	"GREY":    "\x1b[1;37;47m",
+type Colour int
+
+const (
+	BLACK Colour = iota
+	RED
+	GREEN
+	BROWN
+	BLUE
+	MAGENTA
+	CYAN
+	GREY
+)
+
+var COLOURS = map[Colour]string{
```

Comment:
I'm sorry, I know this was in my example, but actually this map should be private, as we don't need to expose this implementation detail to the client code.